source/javascripts/gradient_fade.js

```diff
@@ -0,0 +1,16 @@
+function init() {
+    // Create the listener function
+    var updateLayout = _.debounce(function(e) {
+        var body = $("body");
+        var distanceY = window.pageYOffset || document.documentElement.scrollTop;
+        if (distanceY > 120) {
+            body.css("background", "white");
+        } else {
+            var alpha = 1 - (distanceY / 120);
+            var gradient = "linear-gradient(209deg, rgba(217, 255, 255, "+alpha+"), rgba(252, 255, 246,  "+alpha+") 1300px, white 1900px)";
+            body.css("background", gradient);
+        }
+    }, 5); // Maximum run of once per 5 milliseconds
```

Comment:
Also stuff like this really doesn't need a 5ms resolution IMO. Could probably get away with a much coarser duration like 100ms, 250ms, or something.